<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageToPDF</class>
 <widget class="QMainWindow" name="ImageToPDF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>531</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SITPDF</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagetopdf.qrc">
    <normaloff>:/itp/convert.png</normaloff>:/itp/convert.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="mLoadImages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> Załaduj obraz</string>
    </property>
    <property name="icon">
     <iconset resource="imagetopdf.qrc">
      <normaloff>:/itp/picture.png</normaloff>:/itp/picture.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="mMoveUp">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> Przesuń w górę</string>
    </property>
    <property name="icon">
     <iconset resource="imagetopdf.qrc">
      <normaloff>:/itp/up-arrows.png</normaloff>:/itp/up-arrows.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="mDelete">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> Usuń listę</string>
    </property>
    <property name="icon">
     <iconset resource="imagetopdf.qrc">
      <normaloff>:/itp/remove.png</normaloff>:/itp/remove.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="mMoveDown">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> Przesuń w dół</string>
    </property>
    <property name="icon">
     <iconset resource="imagetopdf.qrc">
      <normaloff>:/itp/down-arrows.png</normaloff>:/itp/down-arrows.png</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>361</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Lista plików</string>
    </property>
    <widget class="QListWidget" name="mListView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>361</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="mClear">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> Wyczyść rekord</string>
    </property>
    <property name="icon">
     <iconset resource="imagetopdf.qrc">
      <normaloff>:/itp/eraser.png</normaloff>:/itp/eraser.png</iconset>
    </property>
   </widget>
   <widget class="QProgressBar" name="mProgress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
}

QProgressBar::chunk {
    background-color: #05B8CC;
    width: 20px;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="mConvertNow">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>480</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  Skonwertuj na PDF</string>
    </property>
    <property name="icon">
     <iconset resource="imagetopdf.qrc">
      <normaloff>:/itp/solutions.png</normaloff>:/itp/solutions.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="mAddFolder">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> Załaduj folder</string>
    </property>
    <property name="icon">
     <iconset resource="imagetopdf.qrc">
      <normaloff>:/itp/folder.png</normaloff>:/itp/folder.png</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>180</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Orientacja pliku PDF</string>
    </property>
    <widget class="QCheckBox" name="mHorizontal">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pozioma</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="mVertical">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pionowa</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Informacja o obrazie</string>
    </property>
    <widget class="QLabel" name="mLoadImage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>321</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="mWHandSizeOnHDD">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="imagetopdf.qrc"/>
 </resources>
 <connections/>
</ui>
